lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   create() {
     this.serverless.cli.log('Creating Stack...');
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+    const stackTags = this.serverless.service.provider.stackTags || { STAGE: this.options.stage };
```

Comment:
Two thoughts here:

1. we need to have some type validation to make sure `provider.stackTags` is in fact an object, otherwise users might face unfriendly error.
2. If we're gonna default to `stage` tag, I believe it should still exist even if the user provided more stack tags.